daemon/cluster/executor/container/adapter.go

```diff
@@ -136,6 +136,12 @@ func (c *containerAdapter) removeNetworks(ctx context.Context) error {
 			if _, ok := err.(*libnetwork.ActiveEndpointsError); ok {
 				continue
 			}
+			if _, ok := err.(libnetwork.ErrNoSuchNetwork); ok {
```

Comment:
No idea about these changes in and of themselves... but it would be better to replace all these checks with a switch.

```go
switch err.(type) {
  case ...:
    continue
}